src/dcqc/tests/tests.py

```diff
@@ -0,0 +1,99 @@
+import hashlib
+
+from dcqc.enums import TestStatus
+from dcqc.file import File
+from dcqc.tests.test_abc import ExternalTestMixin, Process, TestABC
+
+
+class FileExtensionTest(TestABC):
```

Comment:
This is great for now, but I can imagine this module exploding into thousands of lines.  Any ideas for scaling?

Reply:
I'm not sure yet. I almost went with one submodule per tier, but that didn't feel right since it's arbitrary. One submodule file per test seems excessive. I figured that I would decide once we have more tests in front of us, which should hopefully help with deciding how to organize them.